general/argo-workflow-gc/template.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: argo-workflow-enforce-podgc-field
+spec:
+  crd:
+    spec:
+      names:
+        kind: EnforceArgoWorkflowGC
+      validation:
+        openAPIV3Schema:
+          type: object
+          properties:
+            errorMsgAdditionalDetails:
+              type: string
+              default: ""
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package argoworkflowgc
+
+        violation[{"msg": msg}] {
+          input.review.kind.kind == "Workflow"
+          not input.review.object.spec.podGC
+
+          msg := sprintf("podGC field is required in an Argo Workflow manifest. %s", [input.parameters.errorMsgAdditionalDetails])
+        }
```

Comment:
Looks like the violation for `CronWorkflows` is missing.

Reply:
Added!